### ISSUE TITLE
BISM Version 5 not clearly showing Source and Targets

### ISSUE THREAD

**Issue by Guavaq** [microsoft/Analysis-Services#34]
Hi there

Thanks for updating this to allow for the use of AAS Compatibility version 1500.

When using it, the icons are bigger, but I cannot see the Source and Target Servers and databases as shown below.

![image](https://user-images.githubusercontent.com/31457981/66361411-05b22900-e9c2-11e9-8500-2286e6bbeb55.png)

**Comment by aarontx** [comment]
I experience the same issue.

**Comment by christianwade** [comment]
Sorry for the delayed response. I have just released a small increment to V5 that may fix this. I just made the cuttoff line a little lower

This is difficult because there are HDPI issues with Winforms extensions running in VS in general. I raised with the VS team a while ago, but it's been a journey. The customized HDPI fixes for WinForms extensions seem unpredictable, especially when tried on different machines, to put it nicely.
